src/Rules.Framework/Providers/InMemory/DataModel/RuleDataModel.cs

```diff
@@ -4,6 +4,7 @@ namespace Rules.Framework.Providers.InMemory.DataModel
 
     internal sealed class RuleDataModel<TContentType, TConditionType>
     {
+        public bool Active { get; set; } = true;
         public dynamic Content { get; set; }
```

Comment:
A space before next line, to satisfy my OCD :D

Reply:
Added